Add astyle installation instruction (#5306)
geodynamics/aspect: CONTRIBUTING.md

are formatted with this style, so please do not think of it as a road block if
you would like to contribute some code.

### Installing astyle
To correctly indent the code in ASPECT you can use `make indent` or `ninja indent`
depending on the build system you are using. The indenting script needs version 2.04 astyle.

> If you installed deal.ii through candi, the correct astyle if most likely already on your system and in your path.

Reviewer: ```suggestion
If you installed deal.II through candi, the correct astyle may already on your system and in your path.
```
